### PR TITLE
mark docker compose as experimental

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@smarterclayton ptal

File: pkg/cmd/cli/cmd/importer/dockercompose.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ import (
 
 const (
 	dockerComposeLong = `
-Import a Docker Compose file as OpenShift objects
+Import a Docker Compose file as OpenShift objects [EXPERIMENTAL]
```
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/importer/dockercompose.go]: Might be better to put a sentence at the end of the description, more of a blurb and more likely a user will see it.  I think we should go through and have similar sentences for app.json, David's new patch command, and anything else we consider new.

File: pkg/cmd/cli/cmd/importer/dockercompose.go [hunk 2]
```diff
@@ -74,7 +74,7 @@ func NewCmdDockerCompose(fullName string, f *clientcmd.Factory, in io.Reader, ou
 	}
 	cmd := &cobra.Command{
 		Use:     "docker-compose -f COMPOSEFILE",
-		Short:   "Import a docker-compose.yml project into OpenShift",
+		Short:   "Import a docker-compose.yml project into OpenShift [EXPERIMENTAL]",
```
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/importer/dockercompose.go]: I'm ambivalent about this - it ruins the look of the help page and makes the command more obvious (possibly not what we want).  Not sure I like the square brackets either.  Let me sleep on it.

**Comment by bparees** [comment]
ok, switched the long desc to a sentence and tried (experimental) in the short desc.  will bug you in the morning.

**Comment by bparees** [comment]
@smarterclayton morning bump.

File: pkg/cmd/cli/cmd/importer/dockercompose.go [hunk 3]
```diff
@@ -34,7 +34,9 @@ During transformation fields in the compose syntax that are not relevant when ru
 a containerized platform will be ignored and a warning printed.
 
 The command will create objects unless you pass the -o yaml or --as-template flags to generate a
-configuration file for later use.`
+configuration file for later use.
+
+This command is experimental and the behavior is subject to change.`
```
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/importer/dockercompose.go]: `Experimental: This command is under active development and may change without notice` probably

**Comment by bparees** [comment]
@smarterclayton updated.  you ok w/ the `(experimental)` in the short desc?

**Comment by smarterclayton** [comment]
Yeah it's fine - we should add it to app.json as well.

**Comment by bparees** [comment]
yeah i was going to do that but apparently i hadn't rebased and didn't have appjson.go in my repo so i didn't think it was merged yet.  included it now.

**Comment by bparees** [comment]
[merge]

**Comment by smarterclayton** [comment]
Flake https://github.com/openshift/origin/issues/9213 [merge]

**Comment by bparees** [comment]
[merge]

**Comment by bparees** [comment]
[merge]

Ben Parees | OpenShift

**Pull request merged**